app/packages/looker/src/lookers/video.ts

```diff
@@ -228,9 +246,12 @@ export class VideoLooker extends AbstractLooker<VideoState, VideoSample> {
     );
   }
 
-  destroy() {
+  detach() {
```

Comment:
for everyone's benefit - do we have a rule of thumb for what `detach` vs `destroy` is meant to do?

Reply:
The implementations had muddied water. I will add descriptions for them. The intention is a looker can be attached/detached as necessary. A destroyed looker can no longer be used.

The abort controller pattern is set up so that a looker's events can only be aborted once